cypress-tests/cypress/support/commands.js

```diff
@@ -895,6 +916,7 @@ Cypress.Commands.add(
       expect(response.headers["content-type"]).to.include("application/json");
       if (response.status === 200) {
         globalState.set("paymentID", paymentIntentID);
+        globalState.set("connectorId", response.body.connector);
```

Comment:
Why?

This is not needed as globalState **will** always have `connectorId` as it is setup even before the tests start to run (you can check `State.js` for this).

Reply:
For routing there is no where we are get to know which connector the testcases are running. Do you remember we already discussed about this

Reply:
> For routing there is no where we are get to know which connector the testcases are running.

Okay, but how are we getting to know about this? We are just setting the value here

Reply:
Whenever the confirm call happened it get know which connector it went in and handle redirection case will happen

Reply:
Let's keep this change. Can you also do the same for PML? This is very much needed.